Trappist/src/Promact.Trappist.Core/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.DomainModel.Models;
+using Promact.Trappist.Repository.Profile;
+using Promact.Trappist.Web.Models;
+using System.Threading.Tasks;
+
+namespace Promact.Trappist.Core.Controllers
+{
+    [Authorize]
+    [Route("api/profile")]
+    public class ProfileController : Controller
+    {
+        private readonly IProfileRepository _profileRepository;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly UserManager<ApplicationUser> _userManager;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_userManager" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)